website/src/pages/flu/compare-side-by-side.astro

```diff
@@ -0,0 +1,6 @@
+---
+import GenericCompareSideBySidePage from '../../components/views/compareSideBySide/GenericCompareSideBySidePage.astro';
+import { Organisms } from '../../types/Organism';
+---
+
+<GenericCompareSideBySidePage organism={Organisms.flu} hideMutationFilter={true} />
```

Comment:
Will it be a general pattern that we hide the mutation filter for flu (also on the other pages that we will add eventually)? I'm thinking whether it would make more sense to add a `hideMutationFilter` property to the view constants.

Reply:
I added this prop in multiple locations here: https://github.com/GenSpectrum/dashboards/pull/423/files - so I would prefer to leave it like this and refactor in another PR to not be overly confusing